src/System.Threading.AccessControl/tests/ThreadingAclExtensionsTests.cs

```diff
@@ -2,13 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
+using System.Linq;
 using System.Security.AccessControl;
+using System.Security.Principal;
 using Xunit;
 
 namespace System.Threading.Tests
 {
     public static class ThreadingAclExtensionsTests
     {
+        #region Test methods
+
+        #region Existence tests
```

Comment:
FWIW, I find this regions unnecessary clutter.

Reply:
I'll eventually merge these unit tests with the ones from EventWaitHandle and Semaphore pretty soon.
Should I split the unit tests in separate files?